setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup, find_packages
 import sys
 
-version = "1.10"
+version = "1.11dev"
```

Comment:
1.10.2

Reply:
😱

Reply:
```suggestion
version = "1.10.1"
```

Reply:
Good spot @apeltzer !! I think @ewels needs to give you some of his commits now 🙁